src/main/kotlin/test/ktortemplate/core/utils/healthcheck/Health.kt

```diff
@@ -80,39 +89,34 @@ class Health private constructor(val cfg: Configuration) {
             ensureDisableUnambiguous(READY_CHECK_URL)
         }
 
-        private fun getCheck(url: String) = checks.getOrElse(url) {
-            CheckMapBuilder().also {
-                checks += url to it
-            }
-        }
-
         /**
          * Adds a check function to a custom check living at the specified URL
          */
-        fun customCheck(url: String, name: String, check: Check) {
-            getCheck(normalizeURL(url)).add(name, check)
+        private fun addCheckToUrl(url: String, name: String, check: Check) {
```

Comment:
The old `getCheck` function was one of those getter-like things with side effects, for me that's a bit of a code smell.